src/sentry/models/issueuserviews.py

```diff
@@ -0,0 +1,40 @@
+from django.db import models
+from django.db.models import UniqueConstraint
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import region_silo_model
+from sentry.db.models.base import DefaultFieldsModel
+from sentry.db.models.fields.foreignkey import FlexibleForeignKey
+from sentry.models.savedsearch import SortOptions
+
+
+@region_silo_model
+class IssueUserViews(DefaultFieldsModel):
+    """
+    A model for a user's view of the issue stream
+    """
+
+    __relocation_scope__ = RelocationScope.Organization
```

Comment:
I copied this over from savedsearches, but I'm not exactly sure what `RelocationScope` means.